queries/period_slippage.sql

```diff
@@ -410,17 +410,33 @@ prices as (
         ON intrinsic.hour = tt.hour
         and intrinsic.contract_address = token
 ),
+eth_prices as (
+    select
+        date_trunc('hour', minute) as hour,
+        avg(price) as eth_price
+    from
+        prices."layer1_usd_eth"
```

Comment:
What is the difference between this price and prices.usd? From this table we could use the weth price and then we have less dependencies, right?

Reply:
This table only has eth prices and is effectively more efficient to query (just for eth prices). We probably already have eth prices in our previous table (but without any guarantees).